dhcpv4/bsdp/boot_image.go

```diff
@@ -79,11 +79,10 @@ type BootImage struct {
 }
 
 // ToBytes converts a BootImage to a slice of bytes.
```

Comment:
I am not sure why lint did't warn about the method comment. It should be updated.